src/Validation/PresenceVerifierProvider.php

```diff
@@ -14,6 +15,27 @@ class PresenceVerifierProvider extends ValidationServiceProvider
     protected $defer = true;
 
     /**
+     * Register the validation factory.
+     *
+     * @return void
+     */
+    protected function registerValidationFactory()
```

Comment:
Is it necessary to redeclare this if we are extending `ValidationServiceProvider` ?